### PR TITLE
fix: content script wrapper and safari support

### PR DESCRIPTION
**Pull request by louisgv**
Upgrade to [Plasmo 0.68.3 ](https://github.com/PlasmoHQ/plasmo/releases/tag/v0.68.3) which fix the following issue:

- https://discord.com/channels/946290204443025438/1093635366378029076 : the extension should now work with site that uses raw require (2015s-era artifact) such as https://dsf.sharefile.com/Authentication/Login
- Unblock safari support #20

### CODE REVIEW

File: apps/extension/tailwind.config.js [hunk 1]
```diff
@@ -2,6 +2,5 @@
 module.exports = {
   mode: "jit",
   darkMode: "media",
-  content: ["./**/*.tsx"],
-  plugins: [require("@tailwindcss/line-clamp")]
```
Comment (alexanderatallah) [inline apps/extension/tailwind.config.js:6]: weird, seems to still work without this - just curious, do you know why?

File: apps/extension/package.json [hunk 2]
```diff
@@ -40,7 +40,7 @@
     "@types/uuid": "^9.0.1",
```
Comment (alexanderatallah) [inline apps/extension/package.json:40]: let's remove line-clamp from the deps above?

File: apps/extension/tailwind.config.js (hunk 1 above)
Comment (louisgv) [reply apps/extension/tailwind.config.js:6]: line-clamp plugin is now included by default

File: apps/extension/package.json (hunk 2 above)
Comment (louisgv) [reply apps/extension/package.json:39]: Done!

**Pull request merged**